snippets/csharp/VS_Snippets_VBCSharp/csProgGuideStatements/CS/Statements.cs

```diff
@@ -80,9 +76,6 @@ public static void test()
             System.Console.WriteLine(num); // Output: 6            
             //</Snippet3>
 
-            //<Snippet4>
-            DoWork(num);
-            //</Snippet4>
```

Comment:
While I was cleaning this file, I checked some other snippets if they were referenced. Not referenced ones are removed.